tensorflow_recommenders_addons/dynamic_embedding/python/ops/cuckoo_hashtable_ops.py

```diff
@@ -154,7 +154,7 @@ def _create_resource(self):
 
     if self._device_type == "GPU":
       with ops.device(self._device):
-        table_ref = hkv_ops.tfra_hkv_hash_table_of_tensors(
+        table_ref = hkv_ops.tfra_hkv_hash_table_of_tensors_lru(
```

Comment:
I'm a little confused if the OP name is prefixed a `lru`, how to support the other 4 strategies?

Reply:
For cuckoo, only the default lru is supported. For other strategies, refer to hkv_hashtable_ops.py